addons/netfox/properties/property-snapshot.gd

```diff
@@ -2,15 +2,15 @@ extends Object
 class_name PropertySnapshot
 
 static func extract(properties: Array[PropertyEntry]) -> Dictionary:
-	var result: Dictionary = {}
+	var result = {}
 	for property in properties:
 		result[property.to_string()] = property.get_value()
 	result.make_read_only()
 	return result
 
 static func apply(properties: Dictionary, cache: PropertyCache):
 	for property in properties:
-		var pe: PropertyEntry = cache.get_entry(property)
+		var pe = cache.get_entry(property)
```

Comment:
If there is no static typing here, at least a rename is required of `pe` to `property_entry`

Reply:
Sure, but either in a different PR, or later in this PR's lifecycle